src/helpers/flare.ts

```diff
@@ -3,10 +3,12 @@
  */
 
 import fs from 'fs'
+import path from 'path'
 
 import axios from 'axios'
 import FormData from 'form-data'
 
+import {PROJECT_FILES} from '../commands/lambda/constants'
```

Comment:
A helper function which is dependent on an internal command file?

Reply:
Good catch, I have refactored to avoid this